### PR TITLE
Added in srst2 recipes for 0.1.4.6 and 0.2.0

### PR DESCRIPTION
**Pull request by ericenns**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/srst2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - setuptools
+    - numpy >=1.7.1
+    - scipy >=0.12.0
+
+  run:
+    - python
+    - numpy >=1.7.1
+    - scipy >=0.12.0
+    - bowtie2 >=2.1.0
+    - samtools ==0.1.18
```
Comment (bgruening) [inline recipes/srst2/meta.yaml]: do you need setuptools in your run section?

File: recipes/srst2/0.1.4.6/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - setuptools
+    - numpy >=1.7.1
+    - scipy >=0.12.0
+
+  run:
+    - python
+    - numpy >=1.7.1
+    - scipy >=0.12.0
+    - bowtie2 >=2.1.0
+    - samtools ==0.1.18
```
Comment (bgruening) [inline recipes/srst2/0.1.4.6/meta.yaml]: Maybe this can be removed?

File: recipes/srst2/meta.yaml (hunk 1 above)
Comment (ericenns) [inline recipes/srst2/meta.yaml]: setuptools is just needed to be able to install the python package, from looking at other recipes it only appears to be included in the build section.

File: recipes/srst2/0.1.4.6/meta.yaml (hunk 2 above)
Comment (ericenns) [inline recipes/srst2/0.1.4.6/meta.yaml]: Not sure what your askin. If it is in regards to the 0.1.4.6 recipe, I do need it as we have a galaxy tool for 0.1.4.6 that we would like to continue using until we have time to validate 0.2 for our purposes.

**Pull request merged**